Manager/Doctrine/AccessTokenManager.php

```diff
@@ -34,4 +34,13 @@ public function save(AccessToken $accessToken): void
         $this->entityManager->persist($accessToken);
         $this->entityManager->flush();
     }
+
+    public function clearExpired(): int
+    {
+        return $this->entityManager->createQueryBuilder()
+            ->delete(AccessToken::class, 'at')
+            ->where('at.expiry < CURRENT_TIMESTAMP()')
```

Comment:
@HypeMC I'm not sure how I feel about the usage of `CURRENT_TIMESTAMP()` here. Using it means that this feature won't work on all databases supported by Doctrine. On the other hand binding an object (DateTime in this case) via the `setParameter()` method is "broken" (https://github.com/doctrine/orm/pull/7528) in Doctrine 2.6.3 (in our internal application that uses this bundle) and 2.64 which has the fix for it hasn't been tagged yet. What is your opinion on this matter?

Reply:
nvm, CURRENT_TIMESTAMP() is actually a DQL function which gets parsed properly.

Reply:
@X-Coder264 Switched to DateTime after all for consistency & cause timecop doesn't work with `CURRENT_TIMESTAMP`.